.github/workflows/php-rector.yml

```diff
@@ -0,0 +1,43 @@
+name: PHP rector
+
+on:
+  push:
+    branches:
+      - master
+      - 2.x
+    paths:
+      - '**.php'
+      - '**composer.json'
+  pull_request:
+    branches:
+      - master
```

Comment:
```suggestion
```
Here as well

Reply:
I saw that too but it's in every workflow. I'll make a separate PR for this.